tests/test_order.rb

```diff
@@ -71,6 +71,172 @@ def test_process_order
     assert_equal order_process['status'], 'processed'
   end
 
+  def test_refund_total_order
+    sdk = Mercadopago::SDK.new(ENV['ACCESS_TOKEN'])
+    result_card_token = sdk.card_token.create(create_card_token_request)
+    order_request = {
+      type: 'online',
+      processing_mode: 'automatic',
+      total_amount: '1000.00',
+      external_reference: 'ext_ref_1234',
+      transactions: {
+        payments: [
+          {
+            amount: '1000.00',
+            payment_method: {
+              id: 'master',
+              type: 'credit_card',
+              token: result_card_token[:response]['id'],
+              installments: 1
+            }
+          }
+        ]
+      },
+      payer: {
+        email: 'test_1731350184@testuser.com'
+      }
+    }
+    result_created = sdk.order.create(order_request)
+    order_created = result_created[:response]
+
+    sleep(1)
+    result_process = sdk.order.refund(order_created['id'])
+    order_refunded = result_process[:response]
+    assert_equal 201, result_process[:status]
+    assert_equal order_refunded['status'], 'refunded'
+  end
+
+  def test_refund_partial_order
+    sdk = Mercadopago::SDK.new(ENV['ACCESS_TOKEN'])
+    result_card_token = sdk.card_token.create(create_card_token_request)
+    order_request = {
+      type: 'online',
+      processing_mode: 'automatic',
+      total_amount: '1000.00',
+      external_reference: 'ext_ref_1234',
+      transactions: {
+        payments: [
+          {
+            amount: '1000.00',
+            payment_method: {
+              id: 'master',
+              type: 'credit_card',
+              token: result_card_token[:response]['id'],
+              installments: 1
+            }
+          }
+        ]
+      },
+      payer: {
+        email: 'test_1731350184@testuser.com'
+      }
+    }
+    result_created = sdk.order.create(order_request)
+    order_created = result_created[:response]
+
+    refund_request = {
+      transactions: [
+        id: order_created['transactions']['payments'][0]['id'],
+        amount: '25.00'
+      ]
+    }
+
+    sleep(1)
+    result_process = sdk.order.refund(order_created['id'], refund_data: refund_request)
+    order_refunded = result_process[:response]
+    assert_equal 201, result_process[:status]
+    assert_equal order_refunded['status'], 'processed'
+    assert_equal order_refunded['status_detail'], 'partially_refunded'
+  end
+
+  def test_cancel_order
+    sdk = Mercadopago::SDK.new(ENV['ACCESS_TOKEN'])
+    card_token_object = {
+      card_number: '5031433215406351',
+      expiration_year: 2025,
```

Comment:
[nth] Seria bom aumentar o ano de expiração
```suggestion
      expiration_year: 2030,
```